yateto/codegen/log/generic.py

```diff
@@ -74,9 +80,9 @@ def generate(self, cpp, routineCache, gemm_cfg):
     Ceqspp = self._reduce(d.result, C, CmemLayout)
 
     gemmDescr = gemm.Description(
-      leftTerm = TensorDescription(innerAname, AmemLayout, Aeqspp),
-      rightTerm = TensorDescription(innerBname, BmemLayout, Beqspp),
-      result = TensorDescription(innerCname, CmemLayout, Ceqspp),
+      leftTerm=TensorDescription(innerAname, AmemLayout, Aeqspp, d.leftTerm.is_compute_constant),
+      rightTerm=TensorDescription(innerBname, BmemLayout, Beqspp, d.rightTerm.is_compute_constant),
+      result=TensorDescription(innerCname, CmemLayout, Ceqspp, d.result.is_compute_constant),
```

Comment:
These are named parameters of *Description constructor class*. According PEP8 one doesn't need to use white-spaces here

From PEP8:
*Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function paramete*

```
# Correct:
def complex(real, imag=0.0):
    return magic(r=real, i=imag)

# Wrong:
def complex(real, imag = 0.0):
    return magic(r = real, i = imag)
```

The built-in linter in Pycharm emits warnings at this piece of code. To resolve it, I had to remove white-spaces

Reply:
I can revert it - i.e., add white-spaces - because there are many such warnings in Yateto. If I do this the code will be kinda uniformed but it won't be a proper python style